src/pybind/mgr/dashboard/frontend/src/app/shared/services/formatter.service.ts

```diff
@@ -28,7 +28,14 @@ export class FormatterService {
     }
     const unit = n < 1 ? 0 : Math.floor(Math.log(n) / Math.log(divisor));
     const truncatedFloat = this.truncate(n / Math.pow(divisor, unit), decimals);
-    return truncatedFloat === '' ? '-' : truncatedFloat + units[unit];
+    if (truncatedFloat === '') {
+      return '-';
+    }
+    let result = truncatedFloat;
```

Comment:
Maybe you should rename `truncatedFloat` to `result` in line 31 ;)

Reply:
I was thinking about this during the implementation phase, but decided not to 
 name it as you suggested because it is not the result at this stage.